docs/getting_started.rst

```diff
@@ -1,45 +1,133 @@
-.. _getting_started:
```

Comment:
we might need to keep this, this .. _getting_started: means this document can be referred by this tag.